src/Perlang.Interpreter/Typing/TypesResolvedValidator.cs

```diff
@@ -1,3 +1,4 @@
+#nullable enable
```

Comment:
(Drive-by fix which enables #39 for this file. There were actually some `null`-related issues with it. We should really aim for enabling this for the whole project within the 0.2.0 release cycle.)